app/controllers/registration/ManageRegistrationController.scala

```diff
@@ -293,17 +293,20 @@ class ManageRegistrationController @Inject() (
 
   def updateBiksFutureAction(year: Int, changes: List[BenefitInKindRequest], additive: Boolean)(implicit
     request: AuthenticatedRequest[AnyContent]
-  ): Future[Result] =
-    bikListService
+  ): Future[Result] = {
+    logger.info(s"[updateBiksFutureAction] Starting - year: $year, additive: $additive, changes count: ${changes.size}")
```

Comment:
Do we need this? Will it help in any way with monitoring?

Reply:
I don't think we need that log as well. Cleaning that